docs/content/modules/phase_field/introduction/ExpressionBuilder.md

```diff
@@ -0,0 +1,101 @@
+# Expression Builder
+
+Build FParser expressions using C++ operator overloading
+
+Mixing in the `ExpressionBuilder` class into your MOOSE classes will allow you to build FParser expressions using familiar C++ syntax.
+
+## Documentation
```

Comment:
Use "Introduction" or something similar here, "Documentation" doesn't seem like the right word.